eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -209,34 +209,35 @@ extends:
-            inputs:
-              pwsh: true
-              workingDirectory: $(SdkRepoDirectory)
-              filePath: $(SdkRepoDirectory)/eng/common/scripts/Add-IssueLabels.ps1
-              arguments: >
-                -RepoOwner $(SpecRepoOwner)
-                -RepoName $(SpecRepoName)
-                -IssueNumber "$(System.PullRequest.PullRequestNumber)"
-                -Labels $(BreakingChangeLabel)
-                -AuthToken "$(azuresdk-github-pat)"
 
-          - task: PowerShell@2
-            displayName: Remove label from the spec PR
-            condition: and(eq(variables['Build.Reason'], 'PullRequest'), ne(variables['BreakingChangeLabel'], ''), eq(variables['BreakingChangeLabelAction'], 'remove'))
-            inputs:
-              pwsh: true
-              workingDirectory: $(SdkRepoDirectory)
-              filePath: $(SdkRepoDirectory)/eng/common/scripts/Remove-IssueLabel.ps1
-              arguments: >
-                -RepoOwner $(SpecRepoOwner)
-                -RepoName $(SpecRepoName)
-                -IssueNumber "$(System.PullRequest.PullRequestNumber)"
-                -LabelName $(BreakingChangeLabel)
-                -AuthToken "$(azuresdk-github-pat)"
+          - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+            - task: PowerShell@2
+              displayName: Add label to the spec PR
+              condition: and(eq(variables['Build.Reason'], 'PullRequest'), ne(variables['BreakingChangeLabel'], ''), eq(variables['BreakingChangeLabelAction'], 'add'))
+              inputs:
+                pwsh: true
+                workingDirectory: $(SdkRepoDirectory)
+                filePath: $(SdkRepoDirectory)/eng/common/scripts/Add-IssueLabels.ps1
+                arguments: >
+                  -RepoOwner $(SpecRepoOwner)
+                  -RepoName $(SpecRepoName)
+                  -IssueNumber "$(System.PullRequest.PullRequestNumber)"
+                  -Labels $(BreakingChangeLabel)
+                  -AuthToken "$(azuresdk-github-pat)"
+
+            - task: PowerShell@2
+              displayName: Remove label from the spec PR
+              condition: and(eq(variables['Build.Reason'], 'PullRequest'), ne(variables['BreakingChangeLabel'], ''), eq(variables['BreakingChangeLabelAction'], 'remove'))
+              inputs:
+                pwsh: true
+                workingDirectory: $(SdkRepoDirectory)
+                filePath: $(SdkRepoDirectory)/eng/common/scripts/Remove-IssueLabel.ps1
+                arguments: >
+                  -RepoOwner $(SpecRepoOwner)
+                  -RepoName $(SpecRepoName)
+                  -IssueNumber "$(System.PullRequest.PullRequestNumber)"
+                  -LabelName $(BreakingChangeLabel)
+                  -AuthToken "$(azuresdk-github-pat)"
 
           - ${{ if eq(parameters.SkipPullRequestCreation, false) }}:
```

Comment:
WE should probably also add the internal only conditions for these steps.

Reply:
`SkipPullRequestCreation` is set to `true` in the entry yml file. Should be fine without adding extra conditions.

Reply:
I moved those steps inside the `internal project` condition.